Source/Resources/Content/MSGraph/DeviceManagement/DeviceManagementScripts/OS - Enable-TaskSchedulerHistory.json

```diff
@@ -7,7 +7,7 @@
       }
     }
   ],
-  "description": "Enables the Windows Task Scheduler to display history of task runs. It is disabled by default in Windows. This can help troubleshooting scheduled tasks.",
+  "description": "What is it: Enables the Windows Task Scheduler to display history of task runs. \n Simeon Notes: This setting is disabled by default in Windows.",
```

Comment:
**What does this do?**
Enables the Windows Task Scheduler to display history of task runs. This setting is disabled by default in Windows, but the baseline enables it.

**Why should you use this?**
This can help with troubleshooting scheduled tasks.

**What is the end-user impact?**
None. 

**Learn more**
[Enable Windows task scheduler history](https://medium.com/techygeekshome/enable-windows-task-scheduler-history-996a601a178c)